.pre-commit-config.yaml

```diff
@@ -1,10 +1,12 @@
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: 'v0.9.10'
+    rev: v0.9.10
     hooks:
       - id: ruff
       - id: ruff-format
   - repo: https://github.com/PyCQA/bandit
-    rev: '1.8.3'
+    rev: 1.8.3
     hooks:
-    - id: bandit
+      - id: bandit
+        args: ["-c", "pyproject.toml"]
```

Comment:
this and the next line are necessary for pre-commit to have bandid use pyproject.toml for config. see the comment on pyproject.toml for why we need it.